content/dstu2/Authorization.md

```diff
@@ -0,0 +1,156 @@
+# Authorization Server Client Documentation #
+---------------------------------------------
+
+### Introduction ###
+The Cerner Authorization Server currently supports OAuth 2.0 [1] SMART [2, 3] on FHIR launch workflows. As a client, it will require interaction between the client (your client application), the user, the authorization server, a SMART Launch server and a FHIR resource server.
+
+### Registration ###
+In order for your client application to utilize any FHIR protected resources, your client application must first register. To do this, contact your Cerner representative with the following information, the following fields are **required**.
+
+* Name - Application name, this should match what the name of the app will be in the app store
+* Redirect/Callback URI
+* Email address
+
+You may also provide the following **optional** fields.
+
+* Logo URI 
```

Comment:
I'm of the opinion we should simply require an SVG for the logo.  This will reduce any pains with scaling and presentation across a wide range of devices and concerns around exact dimensions.  By the time this goes mainstream, IE8 should drop from its current 2% of traffic to less than 1%, so SVG is going to be well supported.